Assets/__Scripts/Platforms/IRotatingLights.cs

```diff
@@ -0,0 +1,8 @@
+﻿using UnityEngine;
+using SimpleJSON;
+
+public abstract class IRotatingLights : MonoBehaviour {
```

Comment:
The `I` prefix is for defining [interfaces](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/interfaces/), not abstract classes.

Reply:
Renamed class